packages/nextjs/app/debug/_components/contract/utilsDisplay.tsx

```diff
@@ -13,44 +14,99 @@ type DisplayContent =
   | undefined
   | unknown;
 
+type ResultFontSize = "sm" | "base" | "xs" | "lg" | "xl" | "2xl" | "3xl";
+
 export const displayTxResult = (
   displayContent: DisplayContent | DisplayContent[],
-  asText = false,
+  fontSize: ResultFontSize = "sm",
```

Comment:
lets have the default `fontSize` as "base" (Which is currently followed in `main` branch `DisaplayVariable` component) 

And lets explicitly pass "sm" for all the nested things

Reply:
🫡 https://github.com/ByteAtATime/scaffold-eth-2/commit/6d54d6763a69d19ac5f8568fec5bc6de809a2e69